client/src/api.ts

```diff
@@ -0,0 +1,70 @@
+import useSWR, { responseInterface } from 'swr';
+import fetch from 'unfetch';
```

Comment:
No need since latest Chrome is our target!

Reply:
got it, thanks! I'll take it out